src/components/Highlights.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="highlightsbar">
+    <span id="center-highlight">
+      {{ industryindices[highlightsindex1] }}
+      {{ industrybetas[highlightsindex1] }}
+    </span>
+    <span id="left-highlight"> JSE Allshare Index: {{ jseindex }}</span>
+    <span id="right-highlight">
+      {{ industryindices[highlightsindex2] }}
+      {{ industrybetas[highlightsindex2] }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Highlights',
+  data() {
+    return {
+      industrybetas: ['0.7276 (5%)', '0.9 (-7%)', '1.1 (25%)', '0.68 (-30%)'],
+      industryindices: [
+        'Oil and Gas Index: ',
+        'Basic Materials Index: ',
+        'Industrials Index: ',
+        'Consumer Goods Index: ',
+      ],
+      jseindex: '0.74 (+5%)',
+      highlightsindex1: 1,
+      highlightsindex2: 2,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
For the right, center and left highlight this can all be done a lot cleaner using flexbox https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
I recommend checking it out and reimplementing it using flexbox. 

Float's are very old school and can be quite difficult to work with